recipes/intel_install.rb

```diff
@@ -15,6 +15,17 @@
 # OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
 # limitations under the License.
 
+libnuma = "#{node['cfncluster']['sources_dir']}/libnuma.tar.gz"
```

Comment:
I think this needs to go in the dir where the license is fetched (by intelmpi installer?)

Reply:
let's move it in the `/usr/lib64` folder that contains the `/usr/lib64/libnuma.so.1` library

Reply:
ok updated